en-US/firefox/quantum.lang

```diff
@@ -0,0 +1,147 @@
+## NOTE: Demo page at https://www-demo3.allizom.org/firefox/quantum/
+## URL: https://www-demo3.allizom.org/%LOCALE%/firefox/quantum/
+
+
+# HTML page title
+;2x faster and 30% less memory
+2x faster and 30% less memory
+
+
+# HTML page description
+;Firefox Quantum Browser
+Firefox Quantum Browser
+
+
+;The new Firefox arrives November 14, 2017
+The new Firefox arrives November 14, 2017
+
+
+;<strong>Firefox</strong> Quantum
+<strong>Firefox</strong> Quantum
+
+
+;New. Fast. Fierce.
+New. Fast. Fierce.
+
+
+;Launches November 14, 2017. Sign up to stay in the know and get launch updates.
+Launches November 14, 2017. Sign up to stay in the know and get launch updates.
+
+
+# The link points to /newsletter/firefox/
+;Notify me
+Notify me
+
+
+# The link points to /firefox/channel/desktop/#beta
+;Or try beta
+Or try beta
+
+
+;New <strong>Firefox</strong> Quantum arrives November 14, 2017.
+New <strong>Firefox</strong> Quantum arrives November 14, 2017.
+
+
+# Link points to newsletter signup page /newsletter/firefox/
+;Keep me updated
+Keep me updated
+
+
+;2x faster
+2x faster
+
+
+;Powered by a new, cutting-edge engine, Firefox has <a href="%(url)s">doubled its speed</a> from last year. Because the Internet waits for no one.
+Powered by a new, cutting-edge engine, Firefox has <a href="%(url)s">doubled its speed</a> from last year. Because the Internet waits for no one.
+
+
+;Lean, mean speed machine
+Lean, mean speed machine
+
+
+;Firefox Quantum’s new engine uses <a href="%(url)s">30%% less memory</a> than Chrome, so other programs won’t slow down during browsing. Now that’s a win-win.
+Firefox Quantum’s new engine uses <a href="%(url)s">30%% less memory</a> than Chrome, so other programs won’t slow down during browsing. Now that’s a win-win.
+
+
+;Beautiful, intelligent design
+Beautiful, intelligent design
+
+
+;Bookmarks + Pocket
+Bookmarks + Pocket
+
+
+;New Tab
+New Tab
+
+
+;Screenshots
```

Comment:
We may want to specify if we expect people to translate this or not

Reply:
I think this needs to go to main.lang as a product name.

Reply:
Maybe we shouldn't expose the string, since it's the product name